core/src/test/scala/unit/kafka/server/ZkOffsetCommitTest.scala

```diff
@@ -0,0 +1,116 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one or more
+  * contributor license agreements.  See the NOTICE file distributed with
+  * this work for additional information regarding copyright ownership.
+  * The ASF licenses this file to You under the Apache License, Version 2.0
+  * (the "License"); you may not use this file except in compliance with
+  * the License.  You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package kafka.server
+
+import java.util.Properties
+import kafka.api.{OffsetCommitRequest, OffsetFetchRequest}
+import kafka.common.{OffsetAndMetadata, OffsetMetadataAndError, TopicAndPartition}
+import kafka.consumer.SimpleConsumer
+import kafka.utils.TestUtils._
+import kafka.utils._
+import kafka.zk.ZooKeeperTestHarness
+import org.apache.kafka.common.protocol.Errors
+import org.apache.kafka.common.utils.Time
+import org.junit.Assert._
+import org.junit.{After, Before, Test}
+import scala.collection._
+
+class ZkOffsetCommitTest extends ZooKeeperTestHarness {
```

Comment:
We probably don't need this test, just the unit test you added to `KafkaZkClient` is enough. The reason is that the old ZK based consumers are going to be removed soon.

Reply:
Got it, removed `ZkOffsetCommitTest`